pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ApexOperator.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.apex;
+
+import apex.jorje.data.ast.AssignmentOp;
+import apex.jorje.data.ast.BinaryOp;
+import apex.jorje.data.ast.BooleanOp;
+import apex.jorje.data.ast.PostfixOp;
+import apex.jorje.data.ast.PrefixOp;
+
+public class ApexOperator {
```

Comment:
This class is only used as a container. I think it would be better to move both enums to the top level. They belong in the AST package too.